src/main/steps/applicant1/hub-page/sole/content.ts

```diff
@@ -146,6 +146,27 @@ const en = ({ isDivorce, partner, userCase }: CommonContent) => ({
   moneyAndProperty: `You can use the time to decide how your money and property will be divided. This is dealt with separately to the ${
     isDivorce ? 'divorce application' : 'application to end your civil partnership'
   }. <a class="govuk-link" href="https://www.gov.uk/money-property-when-relationship-ends" target="_blank">Find out about dividing money and property</a>`,
```

Comment:
is this an exception?

Reply:
@benouaer I can make a separate pr after to clean up existing links, as there are quite a few scattered about

Reply:
theres going to be a few where the link is mid sentence, this is why i thought anchor tags were fine to use in the content file.

Reply:
a mid sentence link could become quite messy to separate out

Reply:
exactly, though if both are you think this is cleaner then we can change it and i guess we can have an exception for links in sentences.

Reply:
we can make an exception for mid-sentence links
the thing with these anchor tags, is that they're not like a simple <br> or <p> tag, where you just have a start and end tag. you've got the class, href and target attributes, so it feels like it belongs to a template file

Reply:
it doesn't have to be done in this PR, we can do it gradually as needed